inc/u_cx_at_client.h

```diff
@@ -290,14 +302,29 @@ int32_t uCxAtClientCmdEnd(uCxAtClient_t *pClient);
 void uCxAtClientHandleRx(uCxAtClient_t *pClient);
 
 /**
-  * @brief  Handle AT RX data
+  * @brief  Get last I/O error code
   *
   * If the AT client returns U_CX_ERROR_IO you can call this
   * function to get the underlying IO error code.
   *
   * @param[in]  pClient:   the AT client from uCxAtClientInit().
   * @retval                IO error code.
   */
-int32_t uCxAtGetLastIoError(uCxAtClient_t *pClient);
+int32_t uCxAtClientGetLastIoError(uCxAtClient_t *pClient);
+
+/**
+  * @brief  Set command timeout
+  *
+  *
+  * @param[in]  pClient:   the AT client from uCxAtClientInit().
+  * @param      timeoutMs: timeout in millisec.
+  * @param      permanent: when set to true the timeout will be applied
+  *                        permanently for all succeeding AT commands.
+  *                        When set to false it will be used only for next
+  *                        AT command. After that the timeout will be restored
+  *                        to last permanent timeout (or default timeout).
+  */
+void uCxAtClientSetCommandTimeout(uCxAtClient_t *pClient, int32_t timeoutMs,
```

Comment:
Maybe let this function return the current value?

Reply:
Good idea! It's added now